File: src/test/java/no/utgdev/diy/ex2/Ex2Config.java
package no.utgdev.diy.ex2;

//REMOVE import no.utgdev.diy.annotations.Bean;
//REMOVE import no.utgdev.diy.annotations.Import;

import java.util.Collections;
import java.util.List;

//REMOVE @Import(Ex2Config2.class)
public class Ex2Config {
    //REMOVE @Bean
    public List<String> list() {
        return Collections.emptyList();
    }

    //REMOVE @Bean
    public String string() {
        return "";
    }
}

File: src/test/kotlin/no/utgdev/diy/ex2/Ex2Config.kt
package no.utgdev.diy.ex2

import no.utgdev.diy.annotations.Bean;
import no.utgdev.diy.annotations.Import;

@Import([Ex2Config2::class])
class Ex2Config {
    @Bean
    fun list(): List<String> {
        return emptyList()
    }

    @Bean
    fun string(): String {
        return ""
    }
}